content/3_Silver/Func_Graphs.mdx

```diff
@@ -715,7 +715,105 @@ print(" ".join(str(i + 1) for i in ans))
 
 </PySection>
 </LanguageSection>
+### Solution 3
 
+Without using any adjacency lists `floyd(x)` generates answers for all connected vertices. `fillup(a,b)` fills up the vertices, from `a` to `b`, `b` being the coordinate, whose `ans[b]` has already been calculated `floyd(x)`
```

Comment:
i still don't see my changes in this

Reply:
implemented the changes, looking through your previous commit [here](https://github.com/cpinitiative/usaco-guide/pull/4336#:~:text=the%20other%20pr-,Update%20Func_Graphs.mdx,-Verified)

Also, updated this branch, which was 2 commits behind [here](https://github.com/cpinitiative/usaco-guide/commits/master/#:~:text=Merge%20pull%20request%20%234370%20from%20cpinitiative/dependabot/npm_and_yarn/jose%2D2.0.7)

am i missing something ?